src/Microsoft.AspNetCore.Hosting.Server.Abstractions/IServer.cs

```diff
@@ -19,7 +19,8 @@ public interface IServer : IDisposable
         /// <summary>
         /// Start the server with an HttpApplication.
         /// </summary>
-        /// <param name="application">An instance of <see cref="IHttpApplication"/>.</param>
+        /// <param name="application">An instance of <see cref="IHttpApplication{TContext}"/>.</param>
+        /// <typeparam name="TContext">The context associated with the HttpApplication.</typeparam>
```

Comment:
Not sure where `HttpApplication` came from - is that a type or something? Should it just say `application`?